src/components/Login/index.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import isDefined from "@codewell/is-defined";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Login = ({ onChange, endpoint, ...props }) => {
+  const [value, setValue] = useState({});
+
+  const setError = (error) => {
+    setValue({
+      error: error,
+    });
+  };
+
+  useEffect(() => {
+    axios.get(`${endpoint}/user`)
+      .then((result) => {
+        setValue({
+          loggedIn: true,
+          user: result.data,
+        })
+      })
+      .catch((error) => {
+        if (error.response) {
+          switch (error.response.status) {
+            case 401:
+              return setValue({loggedIn: false});
```

Comment:
Will overwrite the whole state. Maybe this is the purpose? Rather set all fields explicit on a reset e.g. 
```
setValue({
  loggedIn: false,
  user: null,
  error: null,
})
```